test/evaluate.js

```diff
@@ -241,6 +241,43 @@ describe('evaluate', async function () {
     assertPointFieldValue(point, 'total_nodes', '3i')
   })
 
```

Comment:
Small nitpick -- should rename this file from `evaluate.js` to `evaluate.test.js` to make things consistent?